### ISSUE TITLE
Migrate to null safety

### ISSUE THREAD

**Issue by nilsreichardt** [adithyaxx/animated-stream-list#9]
## Description
All dependencies are migrated to null safety. So we should migrate this package to null safety.

## Resources
* [How to migrate Dart packages to null safety](https://youtu.be/eBr5tlumwlg)
* [Announcing Dart null safety beta](https://medium.com/dartlang/announcing-dart-null-safety-beta-87610fee6730)

**Comment by nilsreichardt** [comment]
I created a PR: #10

**Comment by StanevPrime** [comment]
@adithyaxx, when do you plan to merge the related PR and to distribute the null-safety version to pub.dev ?

**Comment by tonyborchert** [comment]
Yeah, that would be great. Seems not much is happening on this repository anymore.

**Comment by hrabkin** [comment]
Waiting as well...
